docs/01_nodeos/02_usage/60_how-to-guides/10_how-to-configure-state-storage.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+This how-to describes configuration of the Nodeos `backing store`. `Nodeos` can now use `chainbase` or `rocksdb` as a backing store for smart contract state.
+   
+# Prerequisites
+Version 2.1 or above of the EOSIO development environment. 
+
+# Parameter Definitions 
+Specify which backing store to use with the `chain_plugin` `--backing-store` argument. This argument sets state storage to either `chainbase`, the default, or `rocksdb`.
+
+Config Options for eosio::chain_plugin:
+
+  --backing-store arg (=chainbase)      The storage for state, chainbase or 
+                                        rocksdb
+  --rocksdb-threads arg (=1)            Number of rocksdb threads for flush and
```

Comment:
Hi Philip,

https://github.com/EOSIO/eos/pull/9548 further improved rocksdb configuration.
Gurpreet has updated your other document. The changes are in the latest rocksdb_nodeos_integration branch
and will be merged into develop.

--rocksdb-threads: default is now set to the number of cores on the machine.

Reply:
Updated